db.json

```
@@ -18476,4 +20333,4 @@
         "type": "pcdsdevices.happi.containers.GateValve",
         "z": 870.97
     }
-}
+}
```

Should we put this newline back?
```suggestion
}

```